wg-naming/recommendations/002-blacklist-whitelist-allowlist-denylist.md

```diff
@@ -0,0 +1,95 @@
+
+From a technical communication perspective, using whitelist/blacklist as a naming convention applies metaphor (and, in turn, unintended meaning) when it isn’t needed. More directly descriptive words like allowlist/denylist would enhance understanding. Allowlist/denylist, or simply using allowed/denied as an entity prefix has the added benefit of being easily translatable to other human languages.
+
+
+This term measures up against the evaluation framework as follows:
+
+**First order concerns**: 
+
+In short: is the term overt or identity specific? **Not quite.**
+
+* Is the term overtly racist? **Maybe**. See [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/) for more. 
+* Is the term overtly sexist, transphobic, or pejorative about a gender identity? **No**
+* Is the term overtly ableist, or pejorative to neurodiverse or disabled people? **No**
+* Is the term overtly homophobic? **No**
+
+
+**Second order concerns**:
+
+In short: is the term ambigously harmful, or is it harmful but not to a specific identity? **Yes**. Once again, see [this article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/)) 
+
+In particular, see the following pull-quote:
+
+> In this context, it is worth examining the origins of the term “blacklist” from the Douglas Harper Etymology Dictionary, which states that its origin and history is:
+
+ > >  n.
+
+ > >  also black-list, black list, “list of persons who have incurred suspicion,” 1610s, from black (adj.), here indicative of disgrace, censure, punishment (attested from 1590s, in black book) + list (n.). Specifically of employers’ list of workers considered troublesome (usually for union activity) is from 1888. As a verb, from 1718. Related: Blacklisted; blacklisting. [32]
+
+> It is notable that the first recorded use of the term occurs at the time of mass enslavement and forced deportation of Africans to work in European-held colonies in the Americas.
+
+While it is not directly attacking specific identities, in other context it does have negative connotations.
+
+* Is the term violent? **Partially**
+* Is the term milataristic? **No**
+
+**Third-order concerns**
+
+* Is the term evocative instead of descriptive? **Partially** 
+* Is the term ambiguous? **Yes. As mentioned in the above description, "black" and "white" have different meanings in different cultures.**
+
+
+## Precedents 
+
+* [“Blacklists” and “whitelists”: a salutary warning concerning the prevalence of racist language in discussions of predatory publishing](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/)
+
+* [IETF Network Working Group: Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html)
```

Comment:
Not a concern, but the `Internet-Drafts` are something that we should not use for ref. The definition from ietf states as quoted below. Please see the link [here](https://www.ietf.org/standards/ids//)
```
Internet-Drafts have no formal status, and are subject to change or removal at any time; therefore they should not be cited or quoted in any formal document. 
```